File: 0812-rotate-string/0812-rotate-string.cpp
class Solution {
public:
    bool rotateString(string s, string goal) {
        if (s.size() != goal.size()) {
            return false;
        }
        for (int i = 0; i <= s.size(); i++) {
            if (s.substr(i, s.size()-1) + s.substr(0, i) == goal) {
                return true;
            }
        }

        return false;
    }
};

File: 0812-rotate-string/0812-rotate-string.py
class Solution:
    def rotateString(self, s: str, goal: str) -> bool:
        if (len(s) != len(goal)):
            return False
        size: int = len(goal)
        for i in range(size):
            if ((s[i:size] + s[0:i]) == goal):
                return True
        return False